wiki/Kiai_time/id.md

```diff
@@ -0,0 +1,21 @@
+stub: true
+tags:
+   - timing section
+   - kiai mode
+   - waktu
+---
+
+# Waktu Kiai
+*Untuk peraturan seputar waktu kiai, lihat: [Ranking Criteria](/wiki/Ranking_Criteria)*
+
+**Kiai Time** (atau *kiai*) adalah istilah yang diberikan untuk bagian waktu "khusus" untuk membantu menekankan bagian dari [beatmap](/wiki/Beatmap). Pemain dapat mengetahui apakah bagian dari beatmap memiliki bagian waktu kiai jika mereka melihat efek visual tambahan di latar belakang.
+
+Saat kiai aktif, semua [objek](/wiki/Hit_object) yang dipukul akan berkedip mengikuti [irama ketukan utama](/wiki/Beatmap_Editor/Timing) dan bintang akan meledak dari kursor saat [hit circle](/wiki/Hit_object/Hit_circle) dipukul. Jika tidak, mereka akan jatuh dari kursor saat [slider](/wiki/Hit_object/Slider) atau [spinner](/Wiki/Hit_object/Spinner) diikuti.
+
+Beatmapping
```

Comment:
```suggestion
## Beatmapping
```

Reply:
there is a missing that supposed to be empty line in line #16

Reply:
ok